de.metas.business/src/main/java/org/adempiere/impexp/ADUserImportProcess.java

```diff
@@ -157,7 +154,9 @@ protected I_I_User retrieveImportRecord(final Properties ctx, final ResultSet rs
 	}
 
 	@Override
-	protected ImportRecordResult importRecord(final IMutable<Object> state, final I_I_User importRecord) throws Exception
+	protected ImportRecordResult importRecord(final IMutable<Object> state,
+			final I_I_User importRecord,
+			final boolean isInsertOnly /* not used, this importer always inserts */ ) throws Exception
```

Comment:
same as further up: a javadoc would be better than this comment

Reply:
done